mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -167,6 +167,10 @@ class MetalsGlobal(
       visit: Member => Boolean
   ): SymbolSearch.Result = {
 
+    def isRelevantWorkspaceSymbol(sym: Symbol): Boolean = {
+      sym.isClass || sym.isModule || sym.isPackageObject || sym.isTrait || sym.isInterface
```

Comment:
```suggestion
      sym.isClass || sym.isModule || sym.isPackageObject || sym.isTrait || sym.isInterface || sym.isValue
```

This should fix the issue with `tests.CompletionLspSuite`, though it seems those values are created artificially, but they do exist from the compiler point of view :thinking: 

Anyway, the test should pass if you add this suggestion.

Reply:
https://github.com/scalameta/metals/actions/runs/3970455998/jobs/6806200455#step:4:1175 -- this is the result of this change. I don't think we want to have such behaviour. Moreover,`isRelevantWorkspaceSymbol` was created to solve this issue in the first place.

Reply:
But then we would want to be able to import values from package objects for example I think. The problem with that test is that `address` can't be imported from the case class, so it's not accessible. We would probably have the same issue if we had an object address inside a case class.

Maybe we can filter out anything that owner is a type?

Reply:
Yes, I do think that's the best way to go about it. 😃